packages/libp2p/.aegir.js

```diff
@@ -24,7 +24,9 @@ export default {
       const peerId = await createEd25519PeerId()
       const libp2p = await createLibp2p({
         connectionManager: {
-          inboundConnectionThreshold: Infinity,
+          inboundConnectionThreshold: 1000,
```

Comment:
I take it yup doesn't accept `Infinity` as a number?

Reply:
Correct